### PR TITLE
Lesson 01 Warmup Exercise

### PR DESCRIPTION
**Pull request by krysgu**
Completed lesson 01 warmup exercise from codingbat.

### CODE REVIEW

**Comment by krysgu** [comment]
Please find lesson 2 exercises as well.

File: students/kristoffer_jonson/lesson2/series.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
+def fibonacci(n):
+    """
+    Return nth value of fibonacci sequence
+    :param n: Requested value in fibonacci sequence
+    """
+    previous = 1
+    previous_previous = 0
+    if n == 0:
+        return previous_previous
+    if n == 1:
```
Comment (natasha-aleksandrova) [inline students/kristoffer_jonson/lesson2/series.py]: when you're testing the same variable, you'll want to use if/elif syntax

File: students/kristoffer_jonson/lesson2/fizz_buzz.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+for i in range(1,100):
```
Comment (natasha-aleksandrova) [inline students/kristoffer_jonson/lesson2/fizz_buzz.py]: you'll want this to be `range(1, 101)` to include 100

**Comment by natasha-aleksandrova** [comment]
@krysgu don't forget to submit on the course website as well please

**Pull request merged**